src/test/java/wooteco/subway/section/SectionServiceTest.java

```diff
@@ -0,0 +1,178 @@
+package wooteco.subway.section;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Deque;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import wooteco.subway.exception.section.InvalidDistanceException;
+import wooteco.subway.exception.station.NotFoundStationException;
+import wooteco.subway.line.dao.LineDao;
+import wooteco.subway.section.dao.SectionDao;
+import wooteco.subway.section.dto.DeleteStationDto;
+import wooteco.subway.section.dto.SectionServiceDto;
+import wooteco.subway.station.Station;
+import wooteco.subway.station.dao.StationDao;
+
+@Sql("classpath:initializeData.sql")
+@SpringBootTest
+public class SectionServiceTest {
+
+    private Long lineId;
+    private Long stationSinSeolId;
+    private Long stationDongMyoId;
+    private Long stationDongDaeMoonId;
+    private int distance;
```

Comment:
```suggestion
    private final Long lineId = 1L;
    private final Long stationSinSeolId = 1L;
    private final Long stationDongMyoId = 2L;
    private final Long stationDongDaeMoonId = 3L;
    private final int distance = 10;
```
테스트마다 할당할 필요는 없는 것 같습니다 :)